src/components/signupform/SignupFormContainer.js

```diff
@@ -0,0 +1,31 @@
+import Link from 'next/link'
+import styles from './signupcontainer.module.css'
+import CustomizableAvatar from '../common/customavatar/CustomizableAvatar'
+import SignupForm from './SignupForm'
+
+function SignupFormContainer() {
+  return (
+    <main>
+      <div className={styles.div}>
+        <CustomizableAvatar
+          height={100}
```

Comment:
It isn't good practice to hard code numbers within the body of your code like this. It's best to save them in a variable with a descriptive name at the top of the file. Its best for readability 

const AVATAR_HEIGHT = 100
```suggestion
          height={AVATAR_HEIGHT}
```